src/utils/calculateDuration.js

```diff
@@ -0,0 +1,9 @@
+import { FINISHED_STATES } from '@/utils/states'
+
+function isFinished(state) {
+  return FINISHED_STATES.includes(state)
+}
+
+export function duration(startTime, endTime, state) {
```

Comment:
I would recommend either naming the function `calculateDuration`, and/or consider having it be the default export (which would require removing curly brackets where you're importing it)

Reply:
Generally a utilities file wouldn't have a default export. So I'd go with the more clear name route. 

However I'd suggest this utility belongs in the `@utils/states.js` since it has to do with states. a utility file name `calculateDuration` is probably too specific.

Reply:
I think putting it in @utils/states.js makes sense too